mungegithub/mungers/milestone-maintainer.go

```diff
@@ -0,0 +1,461 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mungers
+
+import (
+	"errors"
+	"fmt"
+	"math"
+	"strings"
+	"time"
+
+	"k8s.io/kubernetes/pkg/util/sets"
+	"k8s.io/test-infra/mungegithub/features"
+	"k8s.io/test-infra/mungegithub/github"
+	"k8s.io/test-infra/mungegithub/mungers/approvers"
+	c "k8s.io/test-infra/mungegithub/mungers/matchers/comment"
+	"k8s.io/test-infra/mungegithub/mungers/matchers/event"
+	"k8s.io/test-infra/mungegithub/mungers/mungerutil"
+	"k8s.io/test-infra/mungegithub/options"
+
+	githubapi "github.com/google/go-github/github"
+)
+
+const (
+	milestoneNotifierName = "MilestoneNotifier"
+
+	milestoneRemoved          = "Milestone Removed"
+	milestoneLabelsIncomplete = "Milestone Labels Incomplete"
+	milestoneLabelsComplete   = "Milestone Labels Complete"
+
+	// Remove the milestone from non-blocking issues if required labels are not present within 3 days
+	milestoneRemovalPeriod = 3 * day
+
+	// Warn about milestone-required labels every day
+	milestoneWarningPeriod = 1 * day
+
+	priorityCriticalUrgent = "priority/critical-urgent"
+
+	commentDetail = `<details>
+Additional instructions available [here](https://github.com/kubernetes/community/blob/master/contributors/devel/release/issues.md)
+</details>`
+)
+
+var (
+	kindMap = map[string]string{
+		"kind/bug":     "Fixes a bug discovered during the current release.",
+		"kind/feature": "New functionality.",
+		"kind/cleanup": "Adding tests, refactoring, fixing old bugs.",
+	}
+
+	priorityMap = map[string]string{
+		priorityCriticalUrgent:        "Never automatically move out of a release milestone; continually escalate to contributor and SIG through all available channels.",
```

Comment:
Yes, this is intentional.  priorityCriticalUrgent is defined as a constant because it is referenced in the code to detect when an issue should be considered a blocker.